packer-rhel7-ami.json

```diff
@@ -1,46 +1,44 @@
 {
     
-    "builders": [{
-      "type": "amazon-ebs",
-      "access_key": "{{user `aws_access_key`}}",
-      "secret_key": "{{user `aws_secret_key`}}",
-      "region": "{{user `region`}}",
-      "source_ami_filter": {
-        "filters": {
-          "virtualization-type": "hvm",
-          "name": "RHEL-7.0*",
-          "root-device-type": "ebs"
-        },
-        "owners": ["309956199498"],
-        "most_recent": true
+  "builders": [{
+    "type": "amazon-ebs",
+    "access_key": "{{user `aws_access_key`}}",
+    "secret_key": "{{user `aws_secret_key`}}",
+    "region": "{{user `region`}}",
+    "source_ami_filter": {
+      "filters": {
+        "virtualization-type": "hvm",
+        "name": "RHEL-7.0*",
+        "root-device-type": "ebs"
       },
-      "instance_type": "{{user `instance_type`}}",
-      "ssh_username": "{{user `ssh_username`}}",
-      "ssh_pty": "true",
-      "ami_name": "RHEL7-hardened-{{timestamp}}",
-      "ssh_keypair_name": "{{user `keypair_name`}}",
-      "iam_instance_profile": "{{user `iam_instance_profile`}}",
-      "ssh_private_key_file": "{{user `key_file_path`}}",
-      "associate_public_ip_address": true
-      
+      "owners": ["309956199498"],
```

Comment:
@ashish1605 Do we want this value hard-coded or made available via variable?

Reply:
@aabond  This is publicly available on AWS. We can leave this value as hard-coded